olm-catalog/serverless-operator/csv.template.yaml

```diff
@@ -338,7 +338,7 @@ spec:
               containers:
                 - name: knative-openshift-ingress
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:knative-openshift-ingress
```

Comment:
Uhoh. I just realized that my script is wrong... These images are missing from the relatedImages block now :see_no_evil:

Reply:
:smile_cat: 
wanna PR something to master, and I cherry-pick :cherries: that here, @markusthoemmes ?

Reply:
yeah on my TODO for today.